compiler/record-minmax/src/HDF5Importer.cpp

```diff
@@ -62,6 +62,10 @@ DataType toInternalDtype(const H5::DataType &h5_type)
   return DataType::Unknown;
 }
 
+void readTensorData(H5::DataSet &tensor, uint8_t *buffer)
+{
+  tensor.read(buffer, H5::PredType::NATIVE_UINT8);
+}
```

Comment:
may be add empty line after this?

Reply:
There are multiple `readTensorData`, so I placed them together for readability.